src/main/java/org/spongepowered/common/entity/projectile/ProjectileLauncher.java

```diff
@@ -183,6 +186,25 @@ private static void configureThrowable(EntityThrowable entity) {
                 return doLaunch(loc.getExtent(), arrow);
             }
         });
+        registerProjectileLogic(SpectralArrow.class, new SimpleItemLaunchLogic<SpectralArrow>(SpectralArrow.class, Items.ARROW) {
+
+            @Override
+            protected Optional<SpectralArrow> createProjectile(EntityLivingBase source, Location<?> loc) {
+                SpectralArrow arrow = (SpectralArrow) new EntitySpectralArrow(source.world, source);
+                ((EntitySpectralArrow) arrow).shoot(source, source.rotationPitch, source.rotationYaw, 0.0F, 3.0F, 0);
+                return doLaunch(loc.getExtent(), arrow);
+            }
+        });
+        registerProjectileLogic(Arrow.class, new SimpleItemLaunchLogic<Arrow>(Arrow.class, Items.ARROW) {
+
+            @Override
+            protected Optional<Arrow> createProjectile(EntityLivingBase source, Location<?> loc) {
+                SpongeImpl.getLogger().warn("An Arrow will be launched. Arrow is not what you should use, maybe you want to use TippedArrow instead");
```

Comment:
Why register `Arrow.class` at all?

Reply:
Basically it's my big question in this PR, should we allow plugins to launch Arrow and launch a TippedArrow or not letting plugins launch an `Arrow` at all ?

@Faithcaio says that we should support launching Arrows. To me, we should only throw a warning. I just merged the two ideas ;)

Reply:
On one hand you can say it removes the surprise of Arrow not launching anything as it's a super type, on the other hand it's strictly incorrect.

That said, it was a previous breaking change, that had no API contract change to cause a compile error.

I'm ok with it. but it does mask the fact that the normal arrows are actually tipped arrows, or the fact there are even multiple types of arrows, but at least it throws a warning.

Reply:
How is this strictly incorrect? TippedArrow extends Arrow.
Also as in the linked issue. 
As a plugin dev if you want to shoot an arrow your first idea will not be to shoot a TippedArrow instead. This could be explained by javadocs but still.
Imo the fact that there is an AbstractArrow in the impl doesn't matter and we should not expose this in the API.

Reply:
So either we follow the Minecraft implementation and we do not allow launching  `Arrow`s, either we are maintaining the false idea people have for the today's Minecraft, which is: "A classic arrow in my inventory will launch a classic arrow entity" where there is no 'classic arrow entity' but only `TippedArrow`s

I see Sponge as an API representing the actual Minecraft, not hiding fact that the implementation is now different than it used to be.